index.rst

```diff
@@ -92,6 +92,14 @@ For a complete list of our projects, check out `Open Data Kit on Github <https:/
   cygwin
   contributing-tips
   style-guide
+
+
+.. toctree::
+  :hidden:
+  :maxdepth: 2
+  :caption: Developing with ODK
```

Comment:
+1

I've been wondering what to call this section. Good idea.

Reply:
Hmm, I would actually put this in the Collect section. It's not really a development thing necessarily, there are apps that already integrate with Collect that could be used "off the shelf". I would call the section something like "External App Integrations".